pyproject.toml

```diff
@@ -49,33 +49,33 @@ djangorestframework = {git = "https://github.com/encode/django-rest-framework.gi
 djangorestframework-xml = "^2.0.0"
 feedparser = "^6.0.10"
 httplib2 = "^0.22.0"
-igraph = "^0.11.3"
+igraph = "^0.11.6"
 internetarchive = "^4.1.0"
 ipaddress = "^1.0.16"
 itypes = "^1.1.0"
-kombu = "^5.3.2"
-lxml = "^5.1.0"
+kombu = "^5.4.2"
+lxml = "^5.3.0"
 markdown2 = "==2.5.*"
 natsort = "^8.4.0"
 ndg-httpsclient = "^0.5.1"
-networkx = "^3.2.1"
+networkx = "^3.3"
```

Comment:
Has changes, doesn't seem to affect anything
https://networkx.org/documentation/stable/release/release_3.3.html#api-changes